chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/RoomDeletedReducer.kt

```diff
@@ -0,0 +1,18 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.RoomDeleted
+import org.reduxkotlin.reducerForActionType
+
+internal val roomDeletedReducer =
+    reducerForActionType<ChatkitState, RoomDeleted> { state, action ->
+        checkNotNull(state.joinedRoomsState)
+
+            state.with(
+                    JoinedRoomsState(
```

Comment:
actually double indentation is still in a few places

Reply:
👍 have fixed